src/components/comment/ActionDropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Media from '../media/Media';
+import { MenuItem } from '../menu';
+import Checkmark16 from '../../icon/fill/Checkmark16';
+import Pencil16 from '../../icon/fill/Pencil16';
+import Trash16 from '../../icon/fill/Trash16';
+import messages from './messages';
+
+import './ActionDropdown.scss';
+
+export interface ActionDropdownProps {
+    commentId: string;
+    onDelete?: (commentId: string) => void;
+    onEdit?: (commentId: string) => void;
+    onResolve?: (commentId: string) => void;
+}
+
+function ActionDropdown({ commentId, onDelete, onEdit, onResolve }: ActionDropdownProps) {
+    if (!onDelete && !onEdit && !onResolve) {
+        return null;
+    }
+
+    return (
+        <Media.Menu data-testid="Comment-actionDropdown" className="bdl-Comment-actionDropdown">
```

Comment:
SUIT naming is largely concerned with defining hierarchy and scope. Here, we'd want to use `bdl-ActionDropdown` or rename the component to `CommentActionDropdown` and use `bdl-CommentActionDropdown`.

Reply:
I've renamed it to `CommentActionDropdown` with appropriate className.